### PR TITLE
Jashan integrate auth design with networking

### PR DESCRIPTION
**Pull request by JashanpreetSingh-dev**
I have integrated my auth code with the design team's Scenes

### CODE REVIEW

File: Assets/Scripts/Authentication/SignUpUI.cs [hunk 1]
```diff
@@ -26,6 +26,7 @@ public class SignUpUI : MonoBehaviour
     public void CreateAccount()
```
Comment (gabeBaluyut) [inline Assets/Scripts/Authentication/SignUpUI.cs:29]: Add documentation for function

File: Assets/Scripts/Authentication/AccountManager.cs [hunk 2]
```diff
@@ -1,7 +1,9 @@
 
+using System;
 using UnityEngine;
 using PlayFab;
 using PlayFab.ClientModels;
+using UnityEngine.SceneManagement;
 using static UserData;
```
Comment (gabeBaluyut) [inline Assets/Scripts/Authentication/AccountManager.cs:8]: Insert header file for tracking date, version, etc.

File: Assets/Scripts/Authentication/AccountManager.cs [hunk 3]
```diff
@@ -24,8 +26,9 @@ public void Awake()
     /// <param name="Username"> a string </param>
```
Comment (gabeBaluyut) [inline Assets/Scripts/Authentication/AccountManager.cs:43]: Insert function comment

**Review by gabeBaluyut — CHANGES_REQUESTED** [review]
Add header comment for file and functions, otherwise good

**Comment by JashanpreetSingh-dev** [comment]
Making a new pull request

**Pull request closed without merging**